lab_4_retrieval_w_clustering/main.py

```diff
@@ -51,6 +62,10 @@ def set_tokenized_corpus(self, tokenized_corpus: TokenizedCorpus) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not tokenized_corpus:
+            raise ValueError
+        self._corpus = tokenized_corpus
+        self._avg_doc_len = sum(len(paragraph) for paragraph in tokenized_corpus) / len(tokenized_corpus)
```

Comment:
you can use here self._corpus here, but it is optional of course

Reply:
fixed